addons/sale/data/mail_data.xml

```diff
@@ -263,16 +265,22 @@
                 </td>
             </tr>
         </table>
-        % if object.carrier_id:
+        % set delivery_line = object.order_line.filtered('is_delivery')
+        % if object.carrier_id and delivery_line:
+            % if object.user_id.has_group('account.group_show_line_subtotals_tax_excluded'):
+                % set delivery_price = delivery_line.price_reduce_taxexcl
+            % else:
+                % set delivery_price = delivery_line.price_reduce_taxinc
+            % endif
```

Comment:
Btw, isn't there a mechanism in coupon to give "free delivery" ?
I suppose the delivery price would still be shown in that case :thinking:

Reply:
Oh, good catch. That would be a tricky one indeed as the delivery discount is made in another line when a delivery reward  program/coupon is applied...